__tests__/nc-news.test.js

```diff
@@ -58,9 +58,19 @@ describe("ncNews API tests", () => {
           votes: 0,
           article_img_url:
             "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          comment_count: 0,
```

Comment:
this test should be more flexible.  try rewrite it so that it still passes even if comment count is not written in the assertion. HINT - use just matchers like objectContaining.